app/models/card.rb

```diff
@@ -1,2 +1,3 @@
 class Card < ApplicationRecord
+  scope :query, ->(string) { where(arel_table[:name].matches("%#{string}%")) }
```

Comment:
Maybe I'm being paranoid, but can the matches portion be .matches("%#{sanitize_sql_like(string)}%")) ?  At first I was generally thinking of injection, but from some reading it seems it would be possible to inject wildcard chars